src/arrayMethodSort.js

```diff
@@ -1,11 +1,40 @@
 'use strict';
 
-/**
- * Implement method Sort
- */
+function toStringSort(firstElment, seondElement) {
+  const firstStrElement = String(firstElment);
+  const secondStrElement = String(seondElement);
+
+  switch (true) {
```

Comment:
It's the wrong usage of `switch` construction
Again, better to use simple `if` statements here :)

Reply:
ok, i got it